src/store/sync/conversation.ts

```diff
@@ -32,41 +45,61 @@ export const handleModifiedConversationsReducer = (
 		}
 	});
 };
-/* state.conversations = {
-				...state.conversations,
-				[msg.cid]: {
-					...state.conversations[msg.cid],
-					messages: merge(state.conversations[msg.cid].messages, {
-						id: msg.id,
-						parent: msg.l
-					})
-				}
-			}; */
+
+const getNewConversationDate = (
+	messages: Array<ConvMessage>,
+	currentFolder: string,
+	oldDate: number,
+	msg: any
```

Comment:
would it be better to use unknown instead of any?

Reply:
Hi, actually it should be properly typed instead of any or unknown.I have updated it with correct type. Thanks for pointing it out